gnovm/stdlibs/testing/context.gno

```diff
@@ -0,0 +1,21 @@
+package testing
+
+import (
+	"chain"
+)
+
```

Comment:
XXX:

testing calls each test using a new `Context{}` (so that they're running always on a clean state)

same goes for subtests

a test may still call `SetContext` to just alter variables

Reply:
I think we should just run a new gnomachine, it’s not only about context,but about sharing any state (global variable too).

Reply:
Done #2975